src/bin/builder_basics.rs

```diff
@@ -5,56 +5,40 @@
 extern crate gio;
 extern crate gtk;
 
-#[cfg(feature = "gtk_3_10")]
```

Comment:
Why did we do that inside the source code instead of just in Cargo.toml?! :)

Reply:
Young we were.